components/edit-collective/sections/Conversations.js

```diff
@@ -81,7 +81,9 @@ const Conversations = ({ collective }) => {
             isLoading={loading}
             onChange={({ target }) => {
               const updatedCollective = cloneDeep(collective);
-              set(updatedCollective, FEATURE_FLAGS.CONVERSATIONS, target.value);
+              const sections = get(updatedCollective, 'settings.collectivePage.sections');
+              const conversationsSection = sections.find(s => s.section === 'conversations');
+              set(conversationsSection, 'isEnabled', target.value);
```

Comment:
`settings.collectivePage.sections` can be `undefined` if the page has not been customized yet.

But anyway, if the toggle on "Edit collective page" changes the feature status, we should just remove this standalone page.

Reply:
Will do